index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="ko-KR">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>여행은 살아보는 거야 - 에어비앤비</title>
+  </head>
+  <body>
+    <!-- 스킵 네비게이션 -->
+    <a href="/" class="skip-nav"><span>컨텐츠로 바로가기</span></a>
+
+    <!-- 헤더 영역 -->
+    <header class="header">
+      <h1 class="logo">
+        <a href="/" class="logo__link">
+          <img src="./image/Airbnb_Logo_Bélo.svg" alt="에어비앤비" class="logo__imaage" />
+        </a>
+      </h1>
```

Comment:
h1 태그에 a태그, 로고 이미지가 들어가 있는게 조금 어색해 보여요!
h1 태그를 통해 홈페이지의 정보를 알려주고, 로고 이미지의 경우 장식이기 때문에 

1. alt=''" 주기
2. 배경 이미지로 처리하기

위와 같은 방식은 어떨까요??

```suggestion
      <h1 class="sr-only">에어비앤비 홈</h1>
        <a href="/" class="logo__link">
          <img src="./image/Airbnb_Logo_Bélo.svg" alt="" class="logo__image" />
        </a>
```

Reply:
오 좋습니다! 훨씬 나은 것 같아요!